src/main/java/core/basesyntax/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.dao.UserDao;
+import core.basesyntax.dao.UserDaoImpl;
+import core.basesyntax.model.Bet;
+import core.basesyntax.model.User;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    private BetDao betDao = new BetDaoImpl();
+    private UserDao userDao = new UserDaoImpl();
+
+    public void handle() {
+        Scanner scanner = new Scanner(System.in);
+
+        while (true) {
+            String command = scanner.nextLine();
+            if (command.equalsIgnoreCase("q")) {
+                scanner.close();
+                return;
+            }
+            Bet bet = null;
+            User user = null;
+            try {
+                String[] userAndBetData = command.split(" ");
+                String name = userAndBetData[0];
+                int age = Integer.parseInt(userAndBetData[1]);
+                if (age < 18) {
+                    throw new IllegalArgumentException();
+                }
+                int value = Integer.parseInt(userAndBetData[2]);
+                double risk = Double.parseDouble(userAndBetData[3]);
+                bet = new Bet(value, risk);
+                user = new User(name, age);
+            } catch (NumberFormatException | ArrayIndexOutOfBoundsException e) {
+                System.out.println("Please, input correct data");
+                continue;
```

Comment:
I am not sure this is good practice to do it like this. "continue" You do not throw an exception, it is simple sout, so as I understand it should work perfectly without continue. Did you check it?

Reply:
We can get NPE here, because below we use method toString() on user and bet. So we need "continue" to move to new loop iteration.